### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by Tekcoder**

### CODE REVIEW

File: app/models/post.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Post < ApplicationRecord
+    validates :title, presence: true
+    belongs_to :user
```
Comment (samgaco) [inline app/models/post.rb:3]: - [ ] There is an association missing here, related with comments. Think about it :)  

_As it was said in the odin project, post should be able to do [this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/building_with_active_record/post_has_many_comments.png)_

File: README.md [hunk 2]
```diff
@@ -1 +1,16 @@
-# Building-With_Active-Records
\ No newline at end of file
+# Micro-Reddit Project from Microverse
+
+This project puts your knowledge of active record, models, and associations to the tests. You will build an application similar to Reddit (called Micro-Reddit) where a user can create a post and add comments to it.
+
+In this project, you’ll build the data structures necessary to support link submissions and commenting. We won’t build a front end for it because we don’t need to… you can use the Rails console to play around with models without the overhead of making HTTP requests and involving controllers or views.
+
+## Installation
```
Comment (samgaco) [inline README.md]: - [ ] The readmefile can be improved, get some inspiration from the [microverse suggested template](https://github.com/microverseinc/readme-template)

**Review by samgaco — CHANGES_REQUESTED** [review]
Hi @Tekcoder and @KingsleyMcSimon :high_brightness: 

Everything is on point, just two things to work on before moving on. You can check them out below in the comments.

- [ ] Also: Don't forget to use [linters](https://github.com/microverseinc/linters-config/tree/master/ruby) 


Good spirit! :muscle: 
------------------------

* ###### STATUS OF THE PROJECT: REQUESTED CHANGES :stop_sign:

**Review by wathigo — APPROVED** [review]
## Approved :+1:  
### Great work @Tekcoder  and @KingsleyMcSimon  :clap:  

**[OPTIONAL]**
* We should make the PR look more professional by adding a descriptive title and a short summary of accomplished tasks as described [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/). 
Since the previous reviewer forgot to mention this, the requirement has been marked as optional.

**You may proceed to the next project.**

**Good luck :crossed_fingers: 
Simon Wathigo**

**Pull request merged**